python/hsml/model.py

```diff
@@ -427,6 +428,28 @@ def set_tag(self, name: str, value: Union[str, dict]):
         A tag consists of a <name,value> pair. Tag names are unique identifiers across the whole cluster.
         The value of a tag can be any valid json - primitives, arrays or json objects.
 
+        !!! warning
+            This method is deprecated and will be remove in 4.0, please use `add_tag` instead.
```

Comment:
Should we better say "in the next version" instead? In case there's a change in the plan.

Reply:
We can say in the future but I think it is just clearer that way. It is not a very controversial item.